internal/kafka/internal/services/quota/internal/test/kafka_supported_instance_types_config.go

```diff
@@ -0,0 +1,131 @@
+package test
+
+import "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+
+func NewQuotaListTestKafkaSupportedInstanceTypesConfig() *config.KafkaSupportedInstanceTypesConfig {
+	cfg := quotaListKafkaSupportedInstanceTypesConfig
+	return &cfg
+}
+
+func NewAMSTestKafkaSupportedInstanceTypesConfig() *config.KafkaSupportedInstanceTypesConfig {
+	cfg := amsKafkaSupportedInstanceTypesConfig
+	return &cfg
+}
+
+var quotaListKafkaSupportedInstanceTypesConfig = config.KafkaSupportedInstanceTypesConfig{
```

Comment:
Can we add the word `test` somewhere in between this name? It is a package-level variable which is also reachable from other files in this package

Reply:
Not necessary as I see it is in a package named `test`.
Why are we creating our own `test` package for that? It does not seem a common practice in Go

Reply:
It is a shared configuration to be used only for testing. I couldn't find a better name.

Reply:
There is a [mocks](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/tree/main/internal/kafka/test/mocks) package, is it possible to move this to that package instead?

Reply:
that package is an `internal` package of Kafka. If we put this there, it won't be accessible.

Reply:
But this package is also within `internal/kafka` so we should be able to import it in the tests within `internal/kafka` which I assume this is what its going to be used for? 

I'm wondering if we can move everything in this file to the mocks package and follow the mock builder pattern that's there?

Reply:
Oh, you are right. They are in the same root. That would work.

I wouldn't say I like putting everything inside the same folder: I'm more for self-containing code than spaghetti dependencies.
When I look at a test, I expect that everything I need is there and if I change something I use in the test, the worst that can happen is that I break that test. If we put everything in a centralised package, you have to worry that it could be used somewhere in the code every time you change it.

Reply:
The decision seems to be between putting it in the internal package or in this test package.
No strong opinion about one way or the other. Having this `test` package is not very idiomatic of Go as it is usually put together in the same package as non-test code.